src/main/kotlin/com/avail/descriptor/representation/BitField.kt

```diff
@@ -95,8 +95,8 @@ class BitField (
 
 	/**
 	 * The [EnumField] with which this `BitField` is annotated, if any.  This is
-	 * populated by the default [
-	 * ][AbstractDescriptor.printObjectOnAvoidingIndent] mechanism.
+	 * populated by the default
+	 * [&#32;][AbstractDescriptor.printObjectOnAvoidingIndent] mechanism.
```

Comment:
`[printing]`

Reply:
I replaced it with just `[printing]`. Good enough.